css/src/atomics/border.scss

```diff
@@ -0,0 +1,85 @@
+$borders-directions: (
+	'top': 'top',
+	'left': $user-left,
+	'bottom': 'bottom',
+	'right': $user-right
+);
+
+.border {
+	border: $border-width solid $border !important;
+}
+
+.border-none {
+	border: 0 !important;
+}
+
+@include tablet {
+	.border-tablet {
+		border: $border-width solid $border !important;
+	}
+
+	.border-none-tablet {
+		border: 0 !important;
+	}
+}
+
+@include desktop {
+	.border-desktop {
+		border: $border-width solid $border !important;
+	}
+}
+
+@include widescreen {
+	.border-widescreen {
+		border: $border-width solid $border !important;
+	}
+}
+
+// Rules for each side
+@each $border-key, $direction in $borders-directions {
+	.border-#{$border-key} {
+		border-#{$direction}: $border-width solid $border !important;
+	}
+
+	.border-#{$border-key}-none {
+		border-#{$direction}: 0 !important;
+	}
+
+	@include tablet {
+		.border-#{$border-key}-tablet {
+			border-#{$direction}: $border-width solid $border !important;
+		}
+
+		.border-#{$border-key}-none-tablet {
+			border-#{$direction}: 0 !important;
+		}
+	}
+
+	@include desktop {
+		.border-#{$border-key}-desktop {
+			border-#{$direction}: $border-width solid $border !important;
+		}
+	}
+	@include widescreen {
+		.border-#{$border-key}-widescreen {
+			border-#{$direction}: $border-width solid $border !important;
+		}
+	}
+}
+
+// Modifiers
+.border-high-contrast {
+	border: $border-width solid $border-white-high-contrast;
+
+	&-hover:hover {
+		border: $border-width solid $border-yellow-high-contrast;
+	}
+}
+
+.border-radius {
+	border-radius: $border-radius !important;
+
+	&-l {
```

Comment:
is the cool syntax worth it? could we just write it out below?

Reply:
Ooops, didn't mean to commit it. Looks weird :D Updated!